tracing/ops_tracing/export.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2025 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+"""OTLP exporter for the ops-tracing extension."""
+
+from __future__ import annotations
+
+import logging
+import ssl
+import threading
+import time
+import urllib.error
+import urllib.request
+from pathlib import Path
+from typing import Sequence
+
+from opentelemetry.sdk.trace import ReadableSpan
+from opentelemetry.sdk.trace.export import SpanExporter, SpanExportResult
```

Comment:
I don't think we want to have these names public. What about renaming this module to `_export.py`?